zebedee-reader/src/main/java/com/github/onsdigital/zebedee/search/fastText/ClientConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package com.github.onsdigital.zebedee.search.fastText;
+
+/**
+ * Wrapping client connection manager setters to simplify configuration for single host
+ */
+public class ClientConfiguration {
+
+    private Integer maxTotalConnection;
+    private boolean disableRedirectHandling;
+
+    public ClientConfiguration() {
+    }
```

Comment:
Is there a reason not pass in the values into the constructor? Instead of creating an empty object and then setting them.